src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BehaviourCompleteKeyFrameCriteriaData.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Text;
+using RegressionGames.StateRecorder.JsonConverters;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models
+{
+    [Serializable]
+    public class BehaviourCompleteKeyFrameCriteriaData : IKeyFrameCriteriaData
+    {
+        public int apiVersion = SdkApiVersion.VERSION_1;
```

Comment:
This needs a new SdkApiVersion defined with a line comment in `SdkApiVersion` describing when this support was added